ACadSharp.Tests/IO/StressSamplesReader.cs

```diff
@@ -0,0 +1,73 @@
+﻿using ACadSharp.IO;
+using System.IO;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ACadSharp.Tests.IO
+{
+	public class StressSamplesReader : IOTestsBase
```

Comment:
Please remove this class from your PR, I'll merge it in the one I've created because it needs some refactoring

Reply:
It created the branch from your PR, that is my bad :D Should I revert the changes in "UserSamplesReader.cs" too?